doc/user_doc/vic_installation/introduction.md

```diff
@@ -1,50 +1,154 @@
-# Overview of vSphere Integrated Containers Engine for vSphere Administrators
+**VICe Overview for vSphere Admins**
```

Comment:
We seem to be mixing VICe and VIC Engine throughout this doc. I propose we not use VICe and stick with VIC Engine